gson/src/main/java/com/google/gson/ExclusionStrategy.java

```diff
@@ -17,11 +17,8 @@
 package com.google.gson;
 
 /**
- * A strategy (or policy) definition that is used to decide whether or not a field or top-level
- * class should be serialized or deserialized as part of the JSON output/input. For serialization,
```

Comment:
Have moved some of this documentation to `GsonBuilder` because there (or rather in the internal `Excluder`) class this logic is implemented.